gui/sharing/models.py

```diff
@@ -85,7 +85,10 @@ class CIFS_Share(Model):
         verbose_name=_('Allow Guest Access'),
         help_text=_(
             'If true then no password is required to connect to the share. '
-            'Privileges will be those of the guest account.'
+            'Privileges will be those of the guest account. Guest access may '
```

Comment:
This particular sentence might be worded strangely to a user. Perhaps:

```suggestion
            'Privileges belong to the guest account. Guest access may '
```

Reply:
I think it means that privileges will be the same as the guest account.
"may" is the wrong word to use here, it implies permission.  "might" is clearer.  But if it happens every time with those Windows versions, it should just be "is":
```suggestion
            'Privileges are the same as the guest account. Guest access is '
```